src/output/icons.rs

```diff
@@ -372,3 +372,319 @@ pub fn icon_for_file(file: &File<'_>) -> char {
+         '\u{e21e}', // 
+         '\u{e73b}', // 
+         '\u{e603}', // 
+         '\u{e737}', // 
+         '\u{e749}', // 
+         '\u{f489}', // 
+         '\u{f489}', // 
+         '\u{e73b}', // 
+         '\u{e70c}', // 
+         '\u{f17c}', // 
+         '\u{f1c0}', // 
+         '\u{e7c4}', // 
+         '\u{f034}', // 
+         '\u{e600}', // 
+         '\u{e600}', // 
+         '\u{f1c5}', // 
+         '\u{e755}', // 
+         '\u{e769}', // 
+         '\u{f410}', // 
+         '\u{f410}', // 
+         '\u{f410}', // 
+         '\u{f410}', // 
+         '\u{f034}', // 
+         '\u{f410}', // 
+         '\u{f1c5}', // 
+         '\u{f410}', // 
+         '\u{e615}', // 
+         '\u{e275}', // 
+         '\u{e628}', // 
+         '\u{f1c3}', // 
+         '\u{e7ba}', // 
+         '\u{f031}', // 
+         '\u{e61c}', // 
+         '\u{f15c}', // 
+         '\u{f410}', // 
+         '\u{f410}', // 
+         '\u{f410}', // 
+         '\u{f03d}', // 
+         '\u{e62b}', // 
+         '\u{e6a0}', // 
+         '\u{e256}', // 
+         '\u{f001}', // 
+         '\u{f03d}', // 
+         '\u{f1c5}', // 
+         '\u{f17a}', // 
+         '\u{f031}', // 
+         '\u{f031}', // 
+         '\u{f13b}', // 
+         '\u{f1c3}', // 
+         '\u{f1c3}', // 
+         '\u{f121}', // 
+         '\u{f121}', // 
+         '\u{f410}', // 
+         '\u{f481}', // 
+         '\u{f481}', // 
+         '\u{f410}', // 
+         '\u{f489}', // 
+         '\u{f489}', // 
+         '\u{f489}', // 
+         '\u{f410}', // 
+         '\u{f15b}', // 
+         '\u{e5fc}', // 
+         '\u{f1d3}', // 
+         '\u{e7b5}', // 
+         '\u{f115}',  // 
+        '\u{f1f8}', // 
+        '\u{e764}', // 
+        '\u{e615}', // 
+        '\u{f489}', // 
+        '\u{f1d3}', // 
+        '\u{f1d3}', // 
+        '\u{f1d3}', // 
+        '\u{f408}', // 
+        '\u{f1d3}', // 
+        '\u{f1d3}', // 
+        '\u{e21e}', // 
+        '\u{e62b}', // 
+        '\u{e70c}', // 
+        '\u{f489}', // 
+        '\u{e7a8}', // 
+        '\u{e5fc}', // 
+        '\u{e5fc}', // 
+        '\u{f308}', // 
+        '\u{f308}', // 
+        '\u{f179}', // 
+        '\u{f1d3}', // 
+        '\u{e626}', // 
+        '\u{e626}', // 
+        '\u{e256}', // 
+        '\u{e611}', // 
+        '\u{e611}', // 
+        '\u{e611}', // 
+        '\u{e610}', // 
+        '\u{e610}', // 
+        '\u{e610}', // 
+        '\u{f023}', // 
+        '\u{e5fc}', // 
+        '\u{f121}', // 
+        '\u{f179}', // 
+        '\u{f489}', // 
+        '\u{e718}', // 
+        '\u{e71e}', // 
+        '\u{f303}', // 
+        '\u{e73b}', // 
+        '\u{e718}' // 
+        ];
+        for icon in all_icons{
+            print!("{} ",icon);
+        }
+        println!("");
+        println!("The number of all icons :{}",all_icons.len());
+    }
+}
```

Comment:
I'm not sure if this is extensive, or the best solution, might give a false sense that icons where actually "tested" in some way